app/views.py

```diff
@@ -8,84 +8,146 @@
 from app.data.repositories import DecksRepository
 from app.data.repositories import CardsRepository
 
+from app import logger
+
+log = logger.getChild(__name__)
 
 @app.route("/")
 def home():
+
+    log.info("route '/' is called")
+
     if "user" in session:
         user = session["user"]
-        return render_template("index.html", user=user)
 
-    return render_template("index.html")
+        log.info("User is authenticated")
+        log.info("Rendering index.html")
+
+        return render_template("index.html", user=user)
 
+    log.info("Rendering index.html")
 
+    return render_template("index.html")
+    
 # TODO ADD ROLES
 @app.route("/show_users")
 def show_users():
+
+    log.info("route 'show_users' is called")
+
     users = UsersRepository.get_all()
 
+    log.info("Users records are retrieved from database")
+
     info = '<br>'.join([' '.join([str(info) for info in user.get_details()]) for user in users])
 
+    log.info("Rendering data")
```

Comment:
remove log.info("Users records are retrieved from database")
its technical info and log like that will be in get_all() as debug

keep log like 'rendering all users data'